src/main/java/org/truffleruby/language/methods/InternalMethod.java

```diff
@@ -173,6 +178,10 @@ public boolean isBuiltIn() {
         return builtIn;
     }
 
+    public boolean isRefined() {
+        return activeRefinements != null;
+    }
```

Comment:
This sounds a bit confusing as it doesn't mean all refined method will return `true` here, only if they come from lookupMethod(). I would do `getActiveRefinements() != null` in callers instead.

Reply:
Part of https://github.com/oracle/truffleruby/pull/2039/commits/6b515da1daaa15c12b505ac0916f404f76b3fcf3